packages/social-auth/src/Auth.ts

```diff
@@ -60,4 +60,28 @@ export abstract class Auth {
 
     return { secretKey, name: accountName };
   }
+
+  async getExtendedCredentials(): Promise<{
```

Comment:
it's better to just extend the original `getCredentials` method. The code is essentially a duplicate. it wouldn't hurt to have more properties, so it's not a breaking change either.

Reply:
Good point. 

The only difference is account name - is that alright?

getCredentials: `const accountName = userInfo.email || userInfo.name || this.idpName;`

getExtendedCredentials: `name: userInfo.name`

Reply:
that's fine. just update the code that uses this bit so that this logic is preserved

Reply:
All done now. 
PTAL at https://github.com/trilitech/umami-v2/pull/1626/commits/f86a3b2a03bb60d6753a9b9d6f74d606cb59af26

Not related to this PR, but useOnboardWithSocial.tsx seems identical for desktop & web, are we planning on moving them to packages and reusing?

Reply:
yes, we do have such plans